### ISSUE TITLE
Crash Report Fabric

### ISSUE THREAD

**Issue by Vitokhv** [jayhillx/MysticsBiomes#54]
Minecraft 1.20.1
Fabric 0.16.7
Java 17

![screenshot_16669](https://github.com/user-attachments/assets/c0e81129-a5e2-4d1a-b11c-032858f19589)

![screenshot_16670](https://github.com/user-attachments/assets/0d517ebc-6e65-4888-ae6d-05d37b145a6a)

[minecraft-exported-crash-info-2024-10-21T04-27-52.zip](https://github.com/user-attachments/files/17451668/minecraft-exported-crash-info-2024-10-21T04-27-52.zip)

```
---- Minecraft Crash Report ----
// Who set us up the TNT?

Time: 2024-10-21 04:26:38
Description: Rendering Block Entity

java.lang.NullPointerException: Cannot read field "field_21531" because "$$9" is null
	at knot//net.minecraft.class_837.method_23083(class_837.java:65)
	at knot//net.minecraft.class_837.method_3569(class_837.java:41)
	at knot//net.minecraft.class_824.method_23079(class_824.java:90)
	at knot//net.minecraft.class_824.method_23081(class_824.java:79)
	at knot//net.minecraft.class_824.method_3554(class_824.java:104)
	at knot//net.minecraft.class_824.method_3555(class_824.java:79)
	at knot//net.minecraft.class_761.method_22710(class_761.java:1367)
	at knot//net.minecraft.class_757.method_3188(class_757.java:1110)
	at knot//net.minecraft.class_757.method_3192(class_757.java:880)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1219)
	at knot//net.minecraft.class_310.method_1514(class_310.java:802)
	at knot//net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```

**Comment by jayhillx** [comment]
hey thank you for telling me! this is now fixed in version 3.4.3! :)

**Comment by Vitokhv** [comment]
Thank you, the game no longer crashes when installing items
Very beautiful mod, I hope in the future it will be updated to 1.20.4 or higher than 1.21.1
